src/Components/Blazor/testassets/HostedInAspNet.Client/HostedInAspNet.Client.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Blazor" />
+    <ProjectReference Include="$(RepoRoot)src\Components\Web.JS\Microsoft.AspNetCore.Components.Web.JS.npmproj" ReferenceOutputAssemblies="false" />
```

Comment:
Instead of duplicating this line in many places, could we add it to the definition of what `ReferenceBlazorBuildLocally` does?